stylesheet.css

```diff
@@ -0,0 +1,310 @@
+}
+
+.nav {
+  grid-area: "nav";
+  font-weight: bold;
+  color: #454545;
+  overflow: none;
+}
+
+.nav, .sidebar {
+  background-color: rgba(255,255,255,0.3);
+  padding: 5px;
+}
+
+.nav ul {
+  padding: 10px;
+  margin: 0;
+}
+
+.nav ul li {
+  padding: 10px;
+  margin:  0.5em 0;
+  list-style-type: none;
+}
+
+.nav ul li > img + span {
+  margin-left: 10px;
+}
+
+.nav ul li:hover, #active {
+  background-color: rgba(255, 255, 255, 0.8);
+  border-radius: 20px;
+}
+
+.createNew {
+  background-color: rgba(255, 255, 255, 0.8);
+  height: min-content;
+  display: grid;
+  grid-template-columns: 50px 1fr;
+  grid-template-rows: 1fr 40px;
+  padding: 20px;
+  gap: 20px;
+}
+
+.textBox {
+  background-color: rgba(255, 255, 255, 1);
+  padding: 10px;
+  border-radius: 10px;
+  box-shadow: 0px 5px 10px rgba(98, 27, 85, 0.1);
+}
+
+.textBox > p {
+  margin: 0;
+}
+
+.createNew .button {
+  font-weight: bold;
+  padding: 10px;
+  border-radius: 20px;
+  border: 1px solid rgb(236, 72, 135); 
+}
+
+.button:hover {
+  background-color: rgb(236, 72, 135);
+  color: rgba(255, 255, 255, 1);
+}
+
+.createNew .OptionsAndSave {
+  grid-column: 2;
+  display: grid;
+  grid-template-columns: 1fr min-content;
+}
+
+.tweet .postOptions {
+  font-size: small;
+  margin-top: 20px;
+  display: grid;
+  gap: 10px;
+  grid-template-columns: repeat(auto-fill, minmax(45px, 1fr));
+}
+
+.tweet .postOptions img {
+  height: 20px;
+}
+
+.OptionsAndSave .postType {
+  display: grid;
+  grid-template-columns: repeat(6, min-content);
+  grid-gap: 3px;
+  align-items: start;
+  justify-items: center;
+}
+
+.postType img {
+  height: 20px;
+  border-radius: 100%;
+  padding: 10%;
+  margin: 0 5px;
+}
+
+.postType img:hover {
+  background-color: rgb(255, 206, 226);
+}
+
+.tweet {
+  background-color: rgba(255, 255, 255, 0.8);
+  border-radius: 10px;
+  padding: 20px;
+  margin: 20px;
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: 50px 1fr;
+  box-shadow: 5px 5px 10px rgba(98, 27, 85, 0.1);
+}
+
+.tweet:hover {
+  cursor: pointer;
+
+  background-color: rgba(255, 255, 255, 1);
+}
+
+.profileName + .handle, .handle + .timestamp {
+  margin-left: 10px;
+}
+
+.tweet .timestamp {
+  font-style: italic;
+  color: rgba(98, 27, 85, 0.5);
+  font-size: smaller;
+}
+
+.tweet .handle {
+  color:rgba(236, 72, 135, 0.5);
+}
+
+.tweet .text {
+  margin-top: 1em;
+  line-height: 150%;
+}
+
+
+.tag, .mention, .profileName, .news p, .followSuggestions p {
+  color: rgb(236, 72, 135);
+}
+
+.profileName {
+  font-weight: bold;
+}
+
+.tag:hover, .mention:hover, .handle:hover {
+  color: rgb(132, 30, 69);
+}
+
+.timestamp:hover {
+  text-decoration: underline;
+}
+
+.tag:before, .news p::before {
+  content: "#"
+}
+
+.mention:before, .handle:before, .followSuggestions p::before {
+  content: "@";
+}
+
+.main {
+  grid-area: "main";
+  max-height: 100vh;
+  overflow-x: none;
+  overflow-y: scroll;
+}
+
```

Comment:
Thoughts about adding:   ```.main::-webkit-scrollbar{
    display: none;
}```


This will make your scroll bar invisible and less early 2000s scroll bar looking.

Reply:
Oh, thank you!

Reply:
Added it in!